package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "https://arjunmahishi.com/type-test",
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
```

Comment:
Let's try to minimize the number of 3rd party libraries used. Unless we are borrowing a lot of functionality from it.

Reply:
Okay got it.